SpinCore_pp/ppg/echo.py

```diff
@@ -86,6 +86,11 @@ def run_spin_echo(
                     returned data from previous run or `None` for the first run.
     """
     assert nEchoes == 1, "you must only choose nEchoes=1"
+    desired_p90_us = p90_us
+    prog_p90_us = prog_plen(desired_p90_us)
+    prog_p180_us = prog_plen(2*desired_p90_us)
+    print("You said you want an actual p90 of %d us and an actual p180 of %d"%(desired_p90_us,2*desired_p90_us))
+    print("So I am sending the SC a p90 time of %d and a p180 time of %d"%(prog_p90_us,prog_p180_us)
```

Comment:
This whole added block has problems with both print statements and the intermediate variable that's not necessary. 

Stopping here